source/assets/sass/vendor/typedoc/_highlight.scss

```diff
@@ -1,5 +1,7 @@
+@use '../syntax-accessible';
```

Comment:
I manually added this here (with single quotes) as this file needs access to the extended classes, but I don't think I need to namespace anything, right? Maybe in a future story we can rewrite this to not @extend anything.